tests/backend/test_organization_stats.py

```diff
@@ -0,0 +1,29 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from uuid import uuid4
+
+from tests.backend.common import vlob_create, block_create
+from parsec.core.backend_connection import BackendConnectionError
+
+
+@pytest.mark.trio
+async def test_organization_stats(running_backend, alice_backend_sock, alice_core, bob_core, realm):
+    organization_stats = await alice_core.get_organization_stats()
+    assert organization_stats.users == 3
```

Comment:
Could be better to compare against an `OrganizationStats` object:
```python
assert organization_stats == OrganizationStats(users=3, data_size=0, metadata_size=ANY)
```

Reply:
Done :)